Storage.md

```diff
@@ -0,0 +1,261 @@
+This guide will describe how FullScreenShenanigans projects store and keep track of game information.
+
+### Table of Contents
+1. [ItemsHoldr](#itemsholdr)
+    1. [Local Storage](#local-storage)
+    2. [Getters and Setters](#getters-and-setters)
+    3. [Updating](#updating)
+    4. [Clearing and Defaults](#clearing-and-defaults)
+    5. [Elements](#elements)
+2. [StateHoldr](#stateholdr)
+
+# ItemsHoldr
+
+ItemsHoldr is a versatile container to store and manipulate values in localStorage, and optionally keep an updated HTML container showing these values.
+
+## Local Storage
+
+Local Storage is a browser property where information is saved past session activity.
```

Comment:
This isn't a very good explanation of local storage. "saved past session activity" is unclear; please rephrase.